machines/ml/Makefile

```diff
@@ -40,6 +40,13 @@ deploy: $(SSH_PRIVATE_KEY_FILE)
 		-var=ami_tag=$(AMI_TAG) \
 		-var=ssh_public_key_file=$(SSH_PUBLIC_KEY_FILE)
 
+.PHONY: destroy
+destroy: $(SSH_PRIVATE_KEY_FILE)
+	terraform destroy \
+		-state=$(SECRETS_DIR)/terraform.tfstate \
+		-var-file=$(CONFIG_FILE) \
+		-var=ami_tag=$(AMI_TAG) \
+		-var=ssh_public_key_file=$(SSH_PUBLIC_KEY_FILE)
```

Comment:
You probably don’t wanna destroy the cache disk. Should we exclude it?

Reply:
I was tired of not having an automated way to do this, so I did the simplest I could. If you know how to improve it quickly please let me know (or do it yourself if you have a laptop at hand). I'm just trying to break a lot of changes I did into small pieces :)

Reply:
Created a fresh environment as to check with total freshness. Since I use the aws controller quite interactively, it happens every now and then

Reply:
Ah, OK. I would `terraform taint` the instance and deploy, but makes sense.